src/forms/inputs/checkbox-group.js

```diff
@@ -34,6 +36,7 @@ import {
  * @param {Object} meta - A `redux-form` [meta](http://redux-form.com/6.5.0/docs/api/Field.md/#meta-props) object
  * @param {Array} options - An array of checkbox values (strings, numbers, or key-value pairs)
  * @param {Object} [checkboxInputProps={}] - An object of key-value pairs representing props to pass down to all checkbox inputs
+ * @param {Boolean} [useDropdown] - A boolean that determines whether the checkbox options are displayed in a dropdown container or not (optional, default `false`)
```

Comment:
I think if you update `[useDropdown]` to `[useDropdown=false]`, that will automatically add in the "(optional, default `false`)" into the generated Docs (like `checkboxInputProps` above).